src/utilities/yamlHeadmatter.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function yamlHeadmatter(fields) {
```

Comment:
What is this doing?

Reply:
This build the yaml matter for the fetched README files, I didn't add this I just moved it to an specific file so the `fetchPackagedFiles` script looks cleaner.